openupgrade_scripts/apriori.py

```diff
@@ -76,6 +90,20 @@
     # OCA/survey
     "survey_description": "survey",
     # OCA/...
+    
+    # Tvtmaaddons
+    "to_l10n_vn_qweb_layout": "l10n_vn_common",
+    "to_l10n_vn_account_detail_sheet": "to_account_reports_l10n_vn", #(erponline-enterprise)
```

Comment:
@hieu1211 sửa như này nhé chú
```suggestion
    "to_l10n_vn_account_detail_sheet": "to_account_reports_l10n_vn", # (moved to Viindoo/erponline-enterprise)
```